index.html

```diff
@@ -5,15 +5,167 @@
+
+              <button>
+                  Let's Team Up
+              </button>
+
+          </div>
+
+      </section>
+      
+
+      <section class="middleContent">
+            <div class="midLeft"> 
+                <h2>
+                    About me
+                </h2> <br>
+
+                <p>
+                    I would be a good hire because I'm very teachable. I can adapt to any work environment. Additionally, my education has spanned multiple fields, which has lent itself to a variety of skill sets and competencies that could potentially be an asset to your company, or, at the very least, speak to my ability to learn and pick up new skills. 
+                </p>
+            </div>
+
+            <div class="midRight">
+                <h2>
+                      Things I'm learning
+                </h2> <br>
+
+                <div class="listContainer">
+                    <p>
+                      Semantic HTML
+                    </p>
+                    <p>
+                        CSS
+                    </p>
+                    <p>
+                        less
+                    </p>
+                    <p>
+                        Github
+                    </p>
+                    <p>
+                        Javascript
+                    </p>
+                </div>
+            </div>
+      </section>
+
+
+      <section class="bottomContent">
+          <h2>
+              Work History
+          </h2>
+
+          <div class="tableContainer">
+              <table style="width:100%">
+              <tr>
+                  <th>
+                    Yrs
+                  </th>
+                  <th>
+                    Company
+                  </th>
+                  <th>
+                    Title
+                  </th>
+              </tr>
+
+              <tr class="coloredRow">
+                  <td>
+                    2
+                  </td>
+                  <td>
+                    University of California, Davis
+                  </td>
+                  <td>
+                    Lecturer
+                  </td>
+              </tr>
+
+              <tr>
+                  <td>
+                    3
+                  </td>
+                  <td>
+                    Boston University 
+                  </td>
+                  <td>
+                    Senior Lecturer
+                  </td>
+              </tr>
+
+
+                  <tr class="coloredRow">
+                    <td>
+                      3
+                    </td>
+                    <td>
+                      IELTS 
+                    </td>
+                    <td>
+                      Reading and Writing Examiner
+                    </td>
+                  </tr>
+              </tr>
+              </table>
+          </div>
+      </section>
+
+
+      <footer>
+          <h3>
+            Contact information | josephnevarez12@gmail.com | 5650 Broadway Sacramento, CA 95864
```

Comment:
How would you have handled this footer text? It looks fine on the desktop view, but behaves non-ideally as it scales down. Maybe I should have made the three pieces of info into three boxes and "flex: grow" them?